backend/tracim_backend/lib/core/userworkspace.py

```diff
@@ -59,6 +64,21 @@ class RoleApi(object):
     #         return tested_role in cls.members_read_rights[reader_role]
     #     return False
 
+    def get_user_workspaces_ids(self, user_id: int, min_role: int) -> typing.List[int]:
```

Comment:
Why not use `tracim_backend.models.roles.WorkspaceRoles` type for `min_role` parameter ?

Reply:
good question, we have mix of all mecanism and new (WorkspaceRoles) , as i work with old code, i didn't think about using it.

Reply:
see #1875

Reply:
I created an issue: #1881